cmd/minishift/cmd/start_test.go

```diff
@@ -34,11 +34,12 @@ import (
 
 	"bytes"
 
+	"github.com/minishift/minishift/cmd/testing/cli"
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	"github.com/minishift/minishift/pkg/minikube/tests"
 	"github.com/minishift/minishift/pkg/minishift/clusterup"
 	instanceState "github.com/minishift/minishift/pkg/minishift/config"
-	"github.com/minishift/minishift/pkg/testing/cli"
+	pkgTest "github.com/minishift/minishift/pkg/testing"
```

Comment:
@hferentschik below you put `pkgTesting` when imported this package and here `pkgTest`, can we have it same across?

Reply:
@praveenkumar I am not sure what is your suggestion? Do you mean we should have "pkgTest" when we import "github.com/minishift/minishift/pkg/testing" ? Because now we have testing in `pkg` as well as `cmd`.

Reply:
You guy realize that there is an import conflict. The only reason I do it, is that there are two 'testing' packages imported/needed here.

Reply:
@hferentschik I know about having two testing package which will import conflict here my suggestion was something below. like you have it  in `cmd/testing/cli/harness.go` file for same commit. It just a name nothing else.

```
pkgTesting "github.com/minishift/minishift/pkg/testing"
```